docs/awesome_list.md

```diff
@@ -0,0 +1,95 @@
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/mFmPDyLlj1E?si=kcYOMjBs3rTz5SRs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+### Fully Functioning UI for Autogen (ep.2)
+
+Author:  [Yeyu Huang](https://twitter.com/Yeyu2HUANG)
+
+Links: [Medium](https://levelup.gitconnected.com/autogens-full-function-ui-powered-by-panel-d00ddecc98ee) | [Substack](https://yeyu.substack.com/p/autogens-full-function-ui-powered?r=21aesy&utm_campaign=post&utm_medium=web) | [YouTube](https://youtu.be/9lSaRP9GLCY?si=RonvtyDOLQCpEOOm)
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/9lSaRP9GLCY?si=EyqLx-lomgXxqJ-2" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+## Fleet AI
+
+[Fleet AI](https://twitter.com/fleet_ai) provides high quality embeddings for more than 1000 popular Python libraries including Panel.
+
+### Chat with +1000 Python Libraries
+
+Build a chat bot on top of the Fleet AI embeddings.
+
+Authors: [Andrew Huang](https://twitter.com/IAteAnDrew1) | [Sophia Yang](https://twitter.com/sophiamyang)
+
+Link: [Blog Post](https://blog.holoviz.org/posts/fleet_ai/)
+
+[<img style="width:560px;height:315px" src="../assets/images/panel-chat-fleet-ai.jpg"></img>](https://blog.holoviz.org/posts/fleet_ai/)
+
+## LangChain
+
+### Building a Retrieval Augmented Generation Chatbot
+
+With [endorsement](https://x.com/hwchase17/status/1730990206856437831?s=20) by [Harrison Chase](https://twitter.com/hwchase17) the founder of LangChain.
+
+Authors: [Andrew Huang](https://twitter.com/IAteAnDrew1) | [Sophia Yang](https://twitter.com/sophiamyang)
+
+Links: [Medium](https://sophiamyang.medium.com/building-a-retrieval-augmented-generation-chatbot-d567a24fcd14)
+
+[![Harrison Chase Endorsed RAG App](assets/images/harrison-chase-testimonial.png)](https://sophiamyang.medium.com/building-a-retrieval-augmented-generation-chatbot-d567a24fcd14)
+
+## Mistral
+
+### ChatBot with Mistral and Llama2
+
+Authors: [Andrew Huang](https://twitter.com/IAteAnDrew1) | [Sophia Yang](https://twitter.com/sophiamyang)
+
+Links: [Medium](https://sophiamyang.medium.com/building-ai-chatbots-with-mistral-and-llama2-9c0f5abc296c)
+
+[![ChatBot with Mistral and Llama2](assets/thumbnails/mistral_and_llama.png)](https://sophiamyang.medium.com/building-ai-chatbots-with-mistral-and-llama2-9c0f5abc296c)
+
+## Ragna
+
+[Ragna](https://ragna.chat/) is an open source RAG orchestration framework. The UI is built on top of Panel.
+
+[<img style="height:315px;" src="../assets/images/ragna-logo.png"></img>](https://ragna.chat/)
```

Comment:
I think this is missing.

Reply:
I think `..` might not be needed?

<img width="757" alt="image" src="https://github.com/holoviz-topics/panel-chat-examples/assets/15331990/f78333fc-b3ec-465e-a548-913ede557e57">